README.md

```diff
@@ -96,7 +96,7 @@ Issues that are currenty know about with workarounds. If you want to
 help find a solution that would be appreciated!
 
 ### IE11
-In some IE11 setups there are issues working with it's native HTML
+In some IE11 setups there are issues working with its native HTML
```

Comment:
The issues involving native support occur with Edge. In IE11 the issues stem from using flash playback not supporting fmp4